benchmark_runner/common/ocp_resources/create_odf.py

```diff
@@ -28,10 +28,28 @@ def create_odf(self):
             if resource.endswith('.sh'):
                 # build sgdisks path dynamically
                 if '01_sgdisks.sh' == resource:
-                    sgdisk_list = ''
-                    for disk_name in self.__ibm_blk_disk_name:
-                        sgdisk_list += f'sgdisk --zap-all /dev/{disk_name};'
-                    self.__oc.run(cmd=f'chmod +x {os.path.join(self.__path, resource)}; {self.__path}/./{resource} "{sgdisk_list}"')
+                    count = 0;
+                    node0 = ''
+                    node0_sgdisk = ''
+                    node1 = ''
+                    node1_sgdisk = ''
+                    node2 = ''
+                    node2_sgdisk = ''
+                    for node, disk_ids in self.__worker_disk_ids.items():
+                        if count == 0:
+                            node0 = node
+                            for disk_id in disk_ids:
+                                node0_sgdisk += f'sgdisk --zap-all /dev/disk/by-id/{disk_id};'
+                        elif count == 1:
+                            node1 = node
+                            for disk_id in disk_ids:
+                                node1_sgdisk += f'sgdisk --zap-all /dev/disk/by-id/{disk_id};'
+                        elif count == 2:
+                            node2 = node
+                            for disk_id in disk_ids:
+                                node2_sgdisk += f'sgdisk --zap-all /dev/disk/by-id/{disk_id};'
+                        count += 1
```

Comment:
Please do this as an array (or dict) rather than hard coding exactly three nodes?